app/src/main/java/org/astraea/partitioner/nodeLoadMetric/LoadPoisson.java

```diff
@@ -10,12 +12,15 @@ public LoadPoisson(NodeLoadClient nodeLoadClient) {
     this.nodeLoadClient = nodeLoadClient;
   }
 
-  public synchronized HashMap<String, Double> setAllPoisson() {
+  public synchronized HashMap<String, Double> setAllPoisson(Set<String> existID) {
     HashMap<String, Double> poissonMap = new HashMap<>();
     int lambda = nodeLoadClient.getAvgLoadCount();
+    var x = 0;
     for (Map.Entry<String, Integer> entry : nodeLoadClient.getAllOverLoadCount().entrySet()) {
-      int x = nodeLoadClient.getBinOneCount(entry.getValue());
-      poissonMap.put(entry.getKey(), doPoisson(lambda, x));
+      if (existID.stream().anyMatch(s -> Objects.equals(s, entry.getKey()))) {
+        x = nodeLoadClient.getBinOneCount(entry.getValue());
```

Comment:
`getBinOneCount` is static method, so you should not call it by instance reference